framework/test/e2e/spark-cicd-userdefined-pipeline.e2e.test.ts

```diff
@@ -0,0 +1,108 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * E2E test for SparkCICDPipeline
+ *
+ * @group e2e/processing/spark-cicd-userdefined
```

Comment:
The path of the tests overlap with the default CICD e2e tests. Running `npx jest --group=e2e/processing/spark-cicd` triggers both e2e. Can you change the test path to non-overlapping?